apps/client/src/app/pages/map/components/route/hooks/mapRoute.tsx

```diff
@@ -0,0 +1,42 @@
+import { LngLat } from 'mapbox-gl';
+
+const calculateRoute = async (
```

Comment:
Consider to use axios and make the call in a APIs folder. Also use axios, if you're already using it, ignore this part